libr/bin/format/elf/elf.c

```diff
@@ -75,6 +75,62 @@ static inline int UTX_MUL(ut64 *r, ut64 a, ut64 b) {
 
 #define round_up(a) ((((a) + (4) - (1)) / (4)) * (4))
 
+#define EF_MIPS_ABI_O32		0x00001000  /* O32 ABI.  */
+#define EF_MIPS_ABI_O64		0x00002000  /* O32 extended for 64 bit.  */
+#define EF_MIPS_ABI			0x0000f000
+
+/*
+ * Return non-zero if HDR identifies an MIPS n64 ELF binary.
+ */
+static int  elfn64_check_arch(Elf_(Ehdr) *__h)
+{
+	int __res = 1;
```

Comment:
just use regular names. In macros they are prefixed with `__` probably to avoid collision with regular names that you could have in the callers, but here there is no need for this.